app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,30 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{:width => "100%"}
+  %table.column{:align => "left"}
+    %tr
+      %td
+        %h3
+          = t(".customer_greeting", name: @order.bill_address.firstname)
+        %h4
+          = t(".instructions_html", distributor: @order.distributor.name )
+        %img.float-left{:src => "#{@order.distributor.logo.url(:medium)}"}/
+  %span.clear
```

Comment:
sorry, I dont understand why we need two tables and this span.clear.
Can you clarify? thanks!

Reply:
This `span.clear` wasn't really necessary actually, but these two tables were necessary to follow the mockup requested on #2910